test/e2e/core.go

```diff
@@ -50,11 +50,18 @@ func CoreDump(dir string) {
 		return
 	}
 
-	cmds := []command{
-		{"cat /var/log/kubelet.log", "kubelet"},
-		{"cat /var/log/kube-proxy.log", "kube-proxy"},
-		{"cat /var/log/supervisor/supervisord.log", "supervisord"},
+	cmds := []command{{"cat /var/log/kube-proxy.log", "kube-proxy"}}
+
+	found := isUsingSystemdKubelet(provider, hosts...)
+	if found {
```

Comment:
one more minor nit (sorry): rather than using the variable `found`, maybe just have
```
if isUsingSystemdKubelet(provider, hosts...) {
   ...
} else {
  ...
}
```

similarly for the master.